client/me/billing-history/upcoming-charges-table.jsx

```diff
@@ -15,6 +15,11 @@ module.exports = React.createClass( {
 
 	render: function() {
 		var transactions = null;
+		const emptyTableText = this.translate( 'The upgrades on your account will not renew automatically. To manage your upgrades or enable Auto Renew visit {{link}}My Upgrades{{/link}}.', {
+			components: {
+				link: <a href="/purchases" />
+			}
```

Comment:
Does the linter complain if you inline this?

```js
components: { link: <a href="/purchases" /> }
```

Also, it would be nice to wrap this text so we can read it without scrolling.

```js
const emptyTableText = this.translate(
	'The upgrades on your account will not renew automatically. ' +
	'To manage your upgrades or enable Auto Renew visit {{link}}My Upgrades{{/link}}.',
	{ components: { link: <a href="/purchaes" /> } }
)
```

The wrapping of the text fits our style closer while the inlining is more about my personal preference for readability.